cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1038,11 +1048,7 @@ def _validate_obsm(self):
         if self._is_supported_spatial_assay() is False and obsm_with_x_prefix == 0:
             self.errors.append("At least one embedding in 'obsm' has to have a key with an 'X_' prefix.")
 
-        is_single = (
-            self.adata.uns["spatial"]["is_single"]
-            if "spatial" in self.adata.uns and "is_single" in self.adata.uns["spatial"]
-            else None
-        )
+        is_single = self._is_single()
```

Comment:
@nayib-jose-gloria, I've updated existing code here to reuse `_is_single()`: can you confirm this looks OK?

Reply:
perfect--thanks for updating!